internal/graph/check.go

```diff
@@ -1438,6 +1421,27 @@ func nestedUsersetFastpath(ctx context.Context,
 	return parallelizeRecursiveMatchUserUserset(ctx, usersetFromObject.Values(), req, recursiveCommonData, recursiveMatchUserUserset)
 }
 
+func buildMappedIterator(ctx context.Context, mapperKind tuplemapper.MapperKind, ds storage.RelationshipTupleReader, req *ResolveCheckRequest, allowedUserTypeRestrictions []*openfgav1.RelationReference) (storage.TupleIterator, error) {
+	var iter storage.TupleIterator
+	var erroriter error
```

Comment:
nitpik: erroriter suggests itself is an iterator.  Perhaps err be a better name?
Also, I am confused in to why do we need to assign to iter and erroriter first?  Why not return directly in line 1429?